manifest.yaml

```diff
@@ -278,4 +278,37 @@ arguments:
     schema:
       type:
         - string
+  # HPA fields
+  hpa.enabled:
+    description: Enable HPA for the primary deployment
+    schema:
+      type: boolean
+  hpa.scaleDown.stabilizationWindowSeconds:
+    description: Stabilization window for HPA for scaling down
+    schema:
+      type: number
+  hpa.scaleUp.stabilizationWindowSeconds:
+    description: Stabilization window for HPA for scaling up
+    schema:
+      type: number
+  hpa.cpuUtilization:
+    description: Use CPU utilization for HPA
+    schema:
+      type: number
```

Comment:
So I think perhaps I should change this path here to 
`hpa.metrics.cpu.averageUtilization`

This will more match the existing name path in k8s
https://docs.openshift.com/container-platform/4.11/rest_api/autoscale_apis/horizontalpodautoscaler-autoscaling-v2.html#status-currentmetrics-resource-current

One thing I wanted to do was not enforce that this had to be specified. My thought was that I would add it, ONLY if specified. Teams can always use the override config to add in a more complicated one which was why I was adding +:

WDYT @jkinkead about the override strategy?